draft-ietf-taps-impl.md

```diff
@@ -738,10 +738,11 @@ errors that can be delivered to the application as Soft Errors. These allow the
 
 ## Pooled Connection {#pooled-connections}
 
-For protocols that employ request/response pairs and do not require in-order delivery of the responses, like HTTP, the transport implementation may distribute interactions across several underlying transport connections. 
-For these kinds of protocols, implementations may hide the connection management and only expose a single Connection object and the individual requests/responses as messages.
-These Pooled Connections can use multiple connections or multiple streams of multi-streaming connections between endpoints, as long as all of these satisfy the requirements, and prohibitions specified in the Selection Properties of the Pooled Connection. 
-This enables implementations to realize transparent connection coalescing, connection migration, and to perform per-message endpoint and path selection by choosing among these underlying connections.
+For applications that do not need in-order delivery of Messages, the Transport Services implementation may distribute Messages of a single Connection across several underlying transport connections or multiple streams of multi-streaming connections between endpoints, as long as all of these satisfy the requirements and prohibitions specified via Selection Properties.
```

Comment:
For me, prohibitions is redundant and just makes reading harder 
```suggestion
For applications that do not need in-order delivery of Messages, the Transport Services implementation may distribute Messages of a single Connection across several underlying transport connections or multiple streams of multi-streaming connections between endpoints, as long as all of these satisfy the requirements specified via Selection Properties.
```

Reply:
I think you do need to check the prohibitions also?

Reply:
I would think so as well.

Reply:
Oh … for me "requirements" actually means everything you express by using "require" or "prohibit" … would probably benefit explicitly stating that and making the remainder easier to read.

Reply:
I tried a fix in the text - if it doesn't please suggest something else.

Reply:
I don't see the fix. But a solution might be: ".... as long as all of these satisfy the specified Selection Properties."

Reply:
```suggestion
For applications that do not need in-order delivery of Messages, the Transport Services implementation may distribute Messages of a single Connection across several underlying transport connections or multiple streams of multi-streaming connections between endpoints, as long as all of these satisfy the Selection Properties.
```